discord/colour.py

```diff
@@ -226,5 +226,15 @@ def blurple(cls):
     def greyple(cls):
         """A factory method that returns a :class:`Colour` with a value of ``0x99aab5``."""
         return cls(0x99aab5)
+    
+    @classmethod
+    def darker_greyple(cls):
+        """
+        A factory method that returns a :class:'Colour' with a value of ``0x36393F``.
+        Will appear transparent on Discord's dark theme and be the text colour on Discord's light theme.
```

Comment:
To avoid confusion in the future, maybe you should clarify the version of Discord that it's been tested against. This classmethod could easily become ignored by the devs at d.py and if Discord changed the background colour on their client, this statement would no longer be true.

Reply:
I see the concern, but I believe that adding such a note would be unnecessary. There are other stuff in this library that isn't based on official "data", which doesn't do this (writing a date/version). Doing this would rather make it more of a hassle because it would preferably be updated each update, as seeing an older version would make it seem outdated. When the case would just be that Discord hasn't changed it.

At this point I am extremely confident that changing the color wouldn't really happen and if it did then we would know, because of the fact that this pull request is 2 years old in a week. And during these years I have yet to update the colour ever.

It's also made clear in the documentation that it will "appear transparent on Discord's dark theme", it is in fact not transparent.